src/producer/index.js

```diff
@@ -26,61 +86,40 @@ module.exports = ({
     disconnect: async () => await cluster.disconnect(),
 
     /**
-     * @param {string} topic
-     * @param {Array} messages An array of objects with "key" and "value", example:
+     * @param {ProduceRequest} ProduceRequest
+     * @returns {Promise}
+     *
+     * @typedef {Object} ProduceRequest
```

Comment:
Here it should be `TopicMessage`, right?

Reply:
No. A `TopicMessage` is an object with a string `topic` and an array of messages. The signature for `send` is different, in order to not break backwards compatibility. For `send`, the object also contains `acks`, `timeout` and `compression`.